components/mailbox/src/Mailbox.svelte

```diff
@@ -291,6 +293,17 @@
           currentPage,
         );
       }
+
+      const lastMessage = main
+        ?.querySelector("nylas-email")
+        ?.shadowRoot?.querySelector(".individual-message.last-message");
```

Comment:
Ideally, we could do this in `Email` component but had some issues with it knowing where the top of the document is. I think it has something to do with multiple nested shadow DOMs.

Reply:
Hmm, should the logic that does this in the Email component be removed then? https://github.com/nylas/components/blob/main/components/email/src/Email.svelte#L422

Reply:
Actually not too sure about this, is there any scenario where `Email` would be used on its own? The logic should work in those use cases.

Reply:
Yeah it seems like the functionality Oliver posted should accomplish this. I'd investigate why it's not working.